api/admin.go

```diff
@@ -69,8 +69,9 @@ func (a *API) adminUsers(w http.ResponseWriter, r *http.Request) error {
 	filter := r.URL.Query().Get("filter")
 	namespaceFilter := r.URL.Query().Get("tigris_namespace")
 	createdByFilter := r.URL.Query().Get("created_by")
+	projectFilter := r.URL.Query().Get("tigris_project")
 
-	users, err := models.FindUsersInAudience(ctx, a.db, instanceID, aud, pageParams, sortParams, filter, namespaceFilter, createdByFilter, a.encrypter)
+	users, err := models.FindUsersInAudience(ctx, a.db, instanceID, aud, pageParams, sortParams, filter, namespaceFilter, createdByFilter, projectFilter, a.encrypter)
```

Comment:
This method is becoming a bit unwieldy. Should a generic filter object be better than 4 different use cases?

Reply:
sure I will refactor it when we add next filter.

Reply:
OK, that's fair. Let's capture as an issue, otherwise LGTM!

Reply:
https://github.com/tigrisdata/gotrue/issues/23